Detectors/ZDC/simulation/CMakeLists.txt

```diff
@@ -9,13 +9,23 @@
 # granted to it by virtue of its status as an Intergovernmental Organization
 # or submit itself to any jurisdiction.
 
-o2_add_library(ZDCSimulation
-               SOURCES src/Detector.cxx src/Digitizer.cxx src/SimCondition.cxx
-               src/ZDCSimParam.cxx src/SpatialPhotonResponse.cxx src/Digits2Raw.cxx
-               src/DigitizerTest.cxx
-               PUBLIC_LINK_LIBRARIES O2::SimulationDataFormat O2::ZDCBase
-                                     O2::DataFormatsZDC O2::CCDB O2::SimConfig
-                                     O2::DetectorsRaw O2::Headers)
+if (ONNXRuntime::ONNXRuntime_FOUND)
+        o2_add_library(ZDCSimulation
+                SOURCES src/Detector.cxx src/Digitizer.cxx src/SimCondition.cxx
+                src/ZDCSimParam.cxx src/SpatialPhotonResponse.cxx src/Digits2Raw.cxx
+                src/DigitizerTest.cxx
+                PUBLIC_LINK_LIBRARIES O2::SimulationDataFormat O2::ZDCBase
+                                        O2::DataFormatsZDC O2::CCDB O2::SimConfig
+                                        O2::DetectorsRaw O2::Headers O2::FASTSimulation)
+else ()
+        o2_add_library(ZDCSimulation
+                SOURCES src/Detector.cxx src/Digitizer.cxx src/SimCondition.cxx
+                src/ZDCSimParam.cxx src/SpatialPhotonResponse.cxx src/Digits2Raw.cxx
+                src/DigitizerTest.cxx
+                PUBLIC_LINK_LIBRARIES O2::SimulationDataFormat O2::ZDCBase
+                                        O2::DataFormatsZDC O2::CCDB O2::SimConfig
+                                        O2::DetectorsRaw O2::Headers)
+endif()
```

Comment:
Here I think we can avoid code duplication by constructing the list of sources or list of link libraries before `o2_add_library` (with conditional list append). Or use some kind of CMake generator expression.

Reply:
List is probably a good idea.

Reply:
Fixed in commit [490480f](https://github.com/AliceO2Group/AliceO2/pull/8396/commits/490480f565d3cb2ab5610f1e7d3b8c67801f2b6a)